app/Localization/Locale.php

```diff
@@ -13,6 +15,15 @@ class Locale
         'da',
     ];
 
+    protected $localeToLanguage = [
+        'en' => "English",
```

Comment:
Single quotes, please! I'm trying to install tlint on the project asap so this can be automated.

Reply:
Done.